### ISSUE TITLE
Check kokoro scripts in google3 for JDK choice

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/cloud-opensource-java#1622]
audit the various release scripts and kokoro configs in google3 to check whether they explicitly specify a JDK. java 8 or Java 11, this should be a deliberate choice, not an accident.

**Comment by elharo** [comment]
In the first case we noticed this is set in a Python script:

cl/328599533

I don't know whether this is a pretty standard way of doing this or a one off for the protobuf team.

**Comment by elharo** [comment]
cs doesn't find many examples of this. While there are some, most must be doing something else.

**Comment by elharo** [comment]
Obvious question: how does this repo's kokoro+rapid builds select a JDK? I don't see it in any of our config files, here or on google3, so we're probably picking up a default somewhere.

**Comment by suztomo** [comment]
Right. I don't see JDK selection https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/dependencies/kokoro/release_build.sh .

Finding non-existence of something sounds difficult.
